All tests Added

Ra1ben opened the pull request:
## ChangeLog

1. Added Registration tests
2. Added Authorization tests
3. Added Placing an order tests
4. Added Order with helper test
5. Added Customer feedback tests
6. Added configurations for all tests

Commit:
Add all tests

Commit:
Add all tests 2

Commit:
Add all tests 3

DmytroSanitarskyi reviewed (changes requested):
Тести супер, все працює як треба, єдине що потрібно виправити, це створити Page object і винести туди всі елементи

Ra1ben committed:
add page object

Ra1ben commented:
## 30.06.2023

### ChangeLog

1. Added Registration Page Object
2. Added Authorization Page Object
3. Added Placing an order Page Object
4. Added Order with helper Page Object
5. Added Customer feedback Page Object
6. Changed all tests considering Page Object

DmytroSanitarskyi reviewed (approved):
Тепер все супер 👍
